src/test/kotlin/html4tree/SymlinkReadTest.kt

```
@@ -0,0 +1,152 @@
+            try {
+                Files.createSymbolicLink(symlinkFile, tempDir.resolve("target.txt"))
+            } catch (e: Exception) {
+                return
+            }
+
+            try {
+                java.nio.file.Files.newInputStream(symlinkFile, java.nio.file.StandardOpenOption.READ, java.nio.file.LinkOption.NOFOLLOW_LINKS).bufferedReader().useLines {
+                    it.toList()
+                }
+            } catch (e: Exception) {
+            }
```

detekt (1.23.8)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)